### PR TITLE
Scripts/Spells: Casting druid spell Prowl auto shapeshifts to cat form

### LINKED ISSUE #19742: Core/Script Spell Prowl working incorrectly.
**Description:**

Spell "Prowl" working incorrectly

**Current behaviour:**

When arrive level 16 you learn the spell "Prowl" (ID: 5215) according to the spell description, when activate, you must change to "cat form" (ID:768) and places you in to stealth. At the moment it´s not changing to "cat form", only enter in to stealth

**Expected behaviour:**

when activate the spell "Prowl" must change to the "cat form" and enter in to "stealth".

**Steps to reproduce the problem:**

1. create a Night Elf player/Druid
2. reach level 16
3. activate the spell "Prowl"

**Branch(es):**

Master

**TC rev. hash/commit:** 

7c66e7bbc7a401e7a2d8f2bdcead6531678262ea

**TDB version:**  7.2.0+updates

**Operating system:** Windows 10

Description in wowhead http://www.wowhead.com/spell=5215/prowl

### PR DESCRIPTION
**Pull request by matanshukry**
**Changes proposed:**

-  Transform you into cat when casting druid prowl spell

**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [x] master

**Issues addressed:**

Closes #19742

**Tests performed:**

- [x] It builds
- [x] Tested in-game

**Known issues and TODO list:** (add/remove lines as needed)

### CODE REVIEW

File: sql/updates/world/master/2021_02_14_00_world_druid_prowl.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Add prowl spell script
+
```
Comment (Faq) [inline sql/updates/world/master/2021_02_14_00_world_druid_prowl.sql:2]:
```suggestion
-- Add prowl spell script
```

File: sql/updates/world/master/2021_02_14_00_world_druid_prowl.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_dru_prowl';
```
Comment (Faq) [inline sql/updates/world/master/2021_02_14_00_world_druid_prowl.sql:3]:
```suggestion
-- Add prowl spell script
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_dru_prowl';
```

**Comment by offl** [comment]
Closes #19742
I recommend to check for open issues to close them after PR will be merged(satisfying)

**Comment by illfated** [comment]
Before commenting in the SQL content: Are you planning on doing some major editing in the SQL file, since you have got such massive amount of whitespace in it?

**Comment by matanshukry** [comment]
@illfated the other .sql file is a WIP waiting on other PRs, it wasn't intended to be added at all; Removed.
and thanks!

**Comment by illfated** [comment]
You're very welcome. To me, it just looked like there was some major changes planned for that somewhat sizeable SQL content.

**Comment by illfated** [comment]
PR title suggestion (English syntax/semantics/readability improvement):
**Scripts/Spells: casting druid spell Prowl auto shapeshifts to cat form**
(as usual, this is a suggestion, based on natural English)

**Pull request merged**